src/components/Card/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Image, CardWrapper } from './styled';
+
+const Card = ({title, imageURL, description, publishedDate}) => {
+  return (
+    <CardWrapper className="ui card" data-testid="card-wrapper">
+      <a className="image" href="/">
+        <Image style={{backgroundImage: `url(${imageURL})`}} />
```

Comment:
Why are you using inline styles if Image is a styled component? You can just pass the background image as a prop.
https://styled-components.com/docs/basics#passed-props